### PR TITLE
Event api widget

### PR DESCRIPTION
**Pull request by MGS31**
Before Reviewing:
!! pull down this branch and have a look !! -- you will need to npm install!

What should happen?

- [ ] You should be able to populate all events data by a specific location ie "Toronto", right now this data is hard coded into the call to the API so if you want to test a different location you need to adjust it within the axios.get

- [ ] On the app page you should see a list of events displayed with a title, Event picture, start date and link to the specific ticketmaster page allowing the users to click through to the page to buy tickets or gain further information.

### CODE REVIEW

File: leafsmart-front-end/package.json [hunk 1]
```diff
@@ -36,5 +36,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "proxy": "http://localhost:3000"
```
Comment (b1u3too) [inline leafsmart-front-end/package.json:40]: Proxy is for letting the server know where to look for any HTTP requests that don't have a full address, like `/login`, right? Should this be localhost:3001 where our server's living in development?

File: leafsmart-front-end/src/components/EventsList.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export default function EventListItem(props) {
```
Comment (b1u3too) [inline leafsmart-front-end/src/components/EventsList.js:3]: The filename and the component being different is wigging out my brain a smidge. Could we make them the same? 
(Maybe the <Events /> component holds a bunch of <EventCard />s?)

**Review by b1u3too — COMMENTED** [review]
Left a couple of comments! Let's discuss them together to see if I'm following appropriately this evening!

**Review by b1u3too — APPROVED** [review]
Simply the best! Thank you thank you~

**Review by rachelpr — APPROVED** [review]

**Pull request merged**